### ISSUE TITLE
Samuel Mahjouri Future Ticket Week 4

### ISSUE THREAD

**Issue by iBraskyy** [macddmac/P3PROPHET#22]
Need to Do:

Change Frontend
- [ ] Create html animal info cards with jinja/css
- [ ] Change color of base.html
- [ ] Create a filtering system for types of animal cards
Databases
- [ ] Look into databases

**Comment by 1843496** [comment]
I like how you are changing the frontend especially how it needs a lot of work. As well as how you are using jinja for the animal info cards. This is good work

**Comment by 1855481** [comment]
Hey Sam, thank you for making our website more physically appealing, appearance is everything when it comes to websites. Go front end!

**Comment by macddmac** [comment]
The ideas you have organized here look like they will be a very good addition to the project. With the better formatted html to better our theme as well as the cool usage of cards and such, I think our website will begin to look much more presentable.

**Comment by shekark642** [comment]
good idea changing the homepage to be more fitting towards our goal. Good concept of using story boards
